src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/JSObject.cs

```diff
@@ -21,6 +21,8 @@ public partial class JSObject : IDisposable
         /// <summary>
         /// Checks whether the target object or one of its prototypes has a property with the specified name.
         /// </summary>
+        /// <param name="propertyName">Name of the property</param>
```

Comment:
Suggestion applies to all similar texts:
```suggestion
        /// <param name="propertyName">The name of the property.</param>
```